.travis.yml

```diff
@@ -6,9 +6,10 @@ addons:
     sources:
     - ubuntu-toolchain-r-test
     packages:
-    - libsdl1.2-dev
-    - libsdl-mixer1.2-dev
-    - libsdl-image1.2-dev
+    - libsdl2-dev
+    - libsdl2-mixer-dev
+    - libsdl2-image-dev
+    - libtiff-dev
```

Comment:
Do we need libtiff-dev? If so I guess it was pulled in by some former *-dev package.

Reply:
I expect so as I did not change anything related to tiff-files, still the compilation complaint about missing tiffio.h.